src/CoreShop/Component/Address/Context/RequestBased/CountryContext.php

```diff
@@ -46,7 +51,13 @@ public function __construct(RequestResolverInterface $requestResolver, RequestSt
     public function getCountry()
     {
         try {
-            return $this->getCountryForRequest($this->getMasterRequest());
+            $request = $this->getMasterRequest();
```

Comment:
Why do you need this variable? It is only used in `$this->getCountryForRequest($request)` (after the `if`)

Reply:
Cause it throws an exception if no request is available